themes/mcf/constants/colors.ts

```diff
@@ -0,0 +1,3 @@
+export const mainAppColour = "#082C43";
```

Comment:
Let's have a catch up, we should try to keep colours defined within the tailwind config